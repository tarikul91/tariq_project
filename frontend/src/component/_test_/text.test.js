import { render, screen,cleanup } from "@testing-library/react";
import Text from "component/text";
import renderer from "react-test-renderer"
afterEach(cleanup)

test('Text test',()=>{
    render(<Text>Some text</Text>)
    expect(screen.getByText('Some text')).toBeInTheDocument()
})
it('text snapshot',()=>{
    const tree = renderer.create(<Text className='text-danger'>Lorem ipsum</Text>).toJSON()
    expect(tree).toMatchSnapshot()
})