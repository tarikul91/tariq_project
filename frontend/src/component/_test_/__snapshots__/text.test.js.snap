// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text snapshot 1`] = `
<p
  className="text-danger"
  data-testid="text"
>
  Lorem ipsum
</p>
`;
